hack/render-sync.sh

```diff
@@ -40,4 +40,6 @@ function rendersync() {
 
 rendersync manual-cluster/performance base/performance default
 rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap/no-mcp
-rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap-cluster/extra-mcp bootstrap/extra-mcp
\ No newline at end of file
+rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap-cluster/extra-mcp bootstrap/extra-mcp
+rendersync --owner-ref none -- base/performance manual-cluster/performance no-ref 
+rendersync --owner-ref none -- base/performance manual-cluster/cpuFrequency default/cpuFrequency 
```

Comment:
don't we want owner reference in this case? it should be only `no-ref` without the reference (hence the name)

Reply:
This syncs the manifests for this test https://github.com/openshift/cluster-node-tuning-operator/blob/master/test/e2e/performanceprofile/functests-render-command/1_render_command/render_test.go#L115 

And it uses no owner references. Probably copy&paste. I believe we should default to having the references most of the time, but I do not think we should fix that as part of this PR.

Reply:
fair enough